tests/test_client.py

```
@@ -0,0 +1,1744 @@
+from unittest import mock
+from typing_extensions import Literal
+
+import httpx
+import pytest
+from respx import MockRouter
+from pydantic import ValidationError
+
+from earn_app import EarnApp, AsyncEarnApp, APIResponseValidationError
+from earn_app._types import Omit
+from earn_app._models import BaseModel, FinalRequestOptions
+from earn_app._exceptions import EarnAppError, APIStatusError, APITimeoutError, APIResponseValidationError
```

Flake8 (7.2.0)

[error] redefinition of unused 'APIResponseValidationError' from line 24

(F811)